### PR TITLE
CSTAR-632: Allow for use of existing DSE tarball rather than download…

### PR DESCRIPTION
**Pull request by csplinter**
…ing new binary

### CODE REVIEW

**Comment by csplinter** [comment]
This is to handle cases with labs that do not have open internet access or do not have access to our vpn. With this PR, we can upload a dse tarball to the cluster and use that to run our tests

**Comment by csplinter** [comment]
@nastra would you mind reviewing?

File: tool/cstar_perf/tool/fab_dse.py [hunk 1]
```diff
@@ -71,6 +75,16 @@ def setup(cfg):
 
         _checkout_dse_branch_and_build_tarball_from_source(branch=revision[4:])
 
+    if use_existing_tarball:
+        if not os.path.exists(use_existing_tarball):
+            raise OSError('Could not find {existing_tarball}, '
```
Comment (nastra) [inline tool/cstar_perf/tool/fab_dse.py]: maybe **FileNotFoundError** instead of **OSError**?

**Comment by nastra** [comment]
+1

File: tool/cstar_perf/tool/fab_dse.py (hunk 1 above)
Comment (csplinter) [inline tool/cstar_perf/tool/fab_dse.py]: **FileNotFoundError** appears to be new to python 3. How about **IOError**?

Comment (nastra) [inline tool/cstar_perf/tool/fab_dse.py]: oh didn't see. Yeah that's fine too. I'm ok with whatever you come up with

**Pull request merged**